flink-tests/src/test/java/org/apache/flink/test/checkpointing/AbstractEventTimeWindowCheckpointingITCase.java

```diff
@@ -115,6 +117,14 @@ public static void stopTestCluster() {
 	@Before
 	public void initStateBackend() throws IOException {
 		switch (stateBackendEnum) {
+			case MEM_ASYNC:
+				this.stateBackend = new AsyncMemoryStateBackend(MAX_MEM_STATE_SIZE);
+				break;
+			case FILE_ASYNC: {
+				String backups = tempFolder.newFolder().getAbsolutePath();
+				this.stateBackend = new AsyncFsStateBackend("file://" + backups);
```

Comment:
That is true. This similar issue should have been there, because this is basically just copy-paste from the `case FILE`. I will fix both.